go.mod

```diff
@@ -50,6 +50,7 @@ require (
 	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
 	github.com/opencontainers/selinux v1.10.0
 	github.com/pelletier/go-toml v1.9.3
+	github.com/pkg/errors v0.9.1
```

Comment:
I think upstream repos are trying to remove this dependency since it's no longer being maintained (see https://github.com/pkg/errors/issues/245)

Reply:
good catch. fixed.